build-system/babel-plugins/babel-plugin-transform-html-template/test/fixtures/transform-assertions/direct/input.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+console.log(html`html usage`);
+console.log(html`html usage
+multiline`);
+console.log(html`<p>
+  <span>html usage</span>
+  <span>more spans</span>
+</p>`);
+console.log(html`<p>
+  <!-- Test Comment to Remove -->
```

Comment:
Add a test for comment before the first element tag.

Reply:
Addressed with 2ecf7a0.